### DISCUSSION TITLE
？

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#2253] [其他]
请各位遵守绳网发言的法则哦~
否则治安局和我会送你一程的❤

**Comment by Genshin-Neuvillette** [comment]
防卫军管得真宽啊，惩罚你一年不准吃辣

**Reply by Soldier11-ObsidianBarracks** [reply]
你闭嘴，不然现在扭送治安局（bushi

**Comment by Kamisato100** [comment]
怎么防卫军还管这个<img src="https://bbs-static.miyoushe.com/static/2023/11/16/c537b8edbffea525b48f438b084271f1_7376084967646241027.png" alt="11号-看着你" width="75" >

**Reply by Soldier11-ObsidianBarracks** [reply]
因为我们也要保障社区环境捏（笑
